Readme.md

```diff
@@ -6,74 +6,96 @@ A Super simple filtering library for IQueryables
+SearchConfiguration.ConfigureSearch(options => 
 {
-    public static City Edmonton = new City { Name = "Edmonton", Country = "Canada", Population = 1000000 };
-    public static City NewYork = new City { Name = "New York", Country = "United States", Population = 20000000 };
+    options.Entity<Order>()
+        .Map(o => o.Customer.ContactName, "customer")
+        .Map(o => o.OrderDetails.Select(od => od.Product.ProductName), "product");
+});
 
-    public IQueryable<Customer> Customers = new List<Customer>()
-    {
-        new Customer() { FirstName = "Joe", LastName = "Smith", City = Edmonton },
-        new Customer() { FirstName = "Jane", LastName = "Smith", City = NewYork },
-        new Customer() { FirstName = "John", LastName = "Doe", City = NewYork }
-    }.AsQueryable();
+// Search for orders by carlos that contain tofu
+query = orders.Search("customer: carlos and product: tofu");
+```
 
-    public void SearchCustomers()
-    {
-        // No Configuration :
-        Customers.Search("LastName: Smith"); // returns Joe and Jane Smith
+### Searching using Comparison Operators
 
-        Customers.Search("Joe").Single(); // returns the single record of Joe Smith
+```csharp
+SearchConfiguration.ConfigureSearch(options => 
+{
+    options.Entity<Product>()
+        .Map(p => p.ProductName, "product")
+        .Map(p => p.UnitPrice, "productPrice")
+        .Map(p => p.UnitsInStock, "productStock");
+});
 
+// Any of (>, >=, < or <=) can be used for comparison on numeric fields
+IQueryable<Order> query = products.Search("productPrice > 10 and productStock <= 5");
 
-        // Configuring searchable fields
-        SearchConfiguration.ConfigureSearch(options =>
-        {
-            options.Entity<Customer>()
-                .AddDefaultSearchField(c => c.FirstName)
-                .AddDefaultSearchField(c => c.LastName)
-                .AddDefaultSearchField(c => c.City.Name)
-                .Map(c => c.FirstName + " " + c.LastName, alias: "name")
-                .Map(c => c.City.Name, alias: "city")
-                .Map(c => c.City.Country, alias: "country")
-                .Map(c => c.City.Population, alias: "citypop")
-                .Map(c => c.City.Name + " " + c.City.Country, alias: "cityinfo");
-        });
+```
 
-        Customers.Search("Name: John OR Edmonton"); // returns John Doe and Joe Smith (in Edmonton)
+### More examples (.NET Fiddle):
+| Name                  | Link                            |
+| --------------------- | ------------------------------- |
+| Basic Search          | https://dotnetfiddle.net/Ox2owv |
+| Navigation Properties | https://dotnetfiddle.net/igWruA |
+| Comparison Operators  | https://dotnetfiddle.net/iaIgqI |
 
-        
-        Customers.Search("name: Joe Smith OR name: John"); // return Joe Smith and John doe
+More examples can be found in the [tests project in this repository](https://github.com/CityofEdmonton/Linq.Search/tree/master/test/Linq.Search.Specification.Tests/Query).
```

Comment:
Is there any example showing the order of operations for this library?

Reply:
Yeah, I'm not really certain what the order of operations will be...

I created an issue for this [here](https://github.com/CityofEdmonton/Linq.Search/issues/2).